src/classification/main.jl

```diff
@@ -268,11 +268,21 @@ function apply_forest(forest::Ensemble{S, T}, features::AbstractVector{S}) where
     end
 end
 
-function apply_forest(forest::Ensemble{S, T}, features::AbstractMatrix{S}) where {S, T}
+function apply_forest(
+        forest::Ensemble{S, T},
+        features::AbstractMatrix{S};
+        use_multithreading = false
+    ) where {S, T}
     N = size(features,1)
     predictions = Array{T}(undef, N)
-    for i in 1:N
-        predictions[i] = apply_forest(forest, features[i, :])
+    if use_multithreading
+        Threads.@threads for i in 1:N
+            predictions[i] = apply_forest(forest, features[i, :])
```

Comment:
```suggestion
            predictions[i] = apply_forest(forest, @view(features[i, :]))
```
You might want to consider using views to avoid cost of allocations, since `apply_forest` doesn't mutate any of its argument and `features` matrix is simply indexed.